From cb4143bc17bf9e2b115792789ca3ab4ed667af08 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Wed, 3 Dec 2014 15:24:07 +0800
Subject: [PATCH 400/602] MA-5929 Update BUILD_ID for l5.0.0_1.0.0-alpha
 release

Update BUILD_ID to 1.0.0-alpha-rc1
Update BUILD_NUMBER to 20141213 as release date

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arm2_6dq/build_id.mk       | 4 ++--
 arm2_6sl/build_id.mk       | 4 ++--
 evk_6sl/build_id.mk        | 4 ++--
 hdmidongle_6dq/build_id.mk | 4 ++--
 imx5x/build_id.mk          | 4 ++--
 sabreauto_6q/build_id.mk   | 4 ++--
 sabreauto_6sx/build_id.mk  | 4 ++--
 sabresd_6dq/build_id.mk    | 4 ++--
 sabresd_6sx/build_id.mk    | 4 ++--
 9 files changed, 18 insertions(+), 18 deletions(-)

diff --git a/arm2_6dq/build_id.mk b/arm2_6dq/build_id.mk
index a9f5b63..5b9d9d2 100644
--- a/arm2_6dq/build_id.mk
+++ b/arm2_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-beta-rc4
-export BUILD_NUMBER=20140815
+export BUILD_ID=1.0.0-alpha-rc1
+export BUILD_NUMBER=20141213
diff --git a/arm2_6sl/build_id.mk b/arm2_6sl/build_id.mk
index a9f5b63..5b9d9d2 100644
--- a/arm2_6sl/build_id.mk
+++ b/arm2_6sl/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-beta-rc4
-export BUILD_NUMBER=20140815
+export BUILD_ID=1.0.0-alpha-rc1
+export BUILD_NUMBER=20141213
diff --git a/evk_6sl/build_id.mk b/evk_6sl/build_id.mk
index a9f5b63..5b9d9d2 100755
--- a/evk_6sl/build_id.mk
+++ b/evk_6sl/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-beta-rc4
-export BUILD_NUMBER=20140815
+export BUILD_ID=1.0.0-alpha-rc1
+export BUILD_NUMBER=20141213
diff --git a/hdmidongle_6dq/build_id.mk b/hdmidongle_6dq/build_id.mk
index a9f5b63..5b9d9d2 100644
--- a/hdmidongle_6dq/build_id.mk
+++ b/hdmidongle_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-beta-rc4
-export BUILD_NUMBER=20140815
+export BUILD_ID=1.0.0-alpha-rc1
+export BUILD_NUMBER=20141213
diff --git a/imx5x/build_id.mk b/imx5x/build_id.mk
index a9f5b63..5b9d9d2 100644
--- a/imx5x/build_id.mk
+++ b/imx5x/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-beta-rc4
-export BUILD_NUMBER=20140815
+export BUILD_ID=1.0.0-alpha-rc1
+export BUILD_NUMBER=20141213
diff --git a/sabreauto_6q/build_id.mk b/sabreauto_6q/build_id.mk
index a9f5b63..5b9d9d2 100644
--- a/sabreauto_6q/build_id.mk
+++ b/sabreauto_6q/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-beta-rc4
-export BUILD_NUMBER=20140815
+export BUILD_ID=1.0.0-alpha-rc1
+export BUILD_NUMBER=20141213
diff --git a/sabreauto_6sx/build_id.mk b/sabreauto_6sx/build_id.mk
index a9f5b63..5b9d9d2 100644
--- a/sabreauto_6sx/build_id.mk
+++ b/sabreauto_6sx/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-beta-rc4
-export BUILD_NUMBER=20140815
+export BUILD_ID=1.0.0-alpha-rc1
+export BUILD_NUMBER=20141213
diff --git a/sabresd_6dq/build_id.mk b/sabresd_6dq/build_id.mk
index a9f5b63..5b9d9d2 100644
--- a/sabresd_6dq/build_id.mk
+++ b/sabresd_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-beta-rc4
-export BUILD_NUMBER=20140815
+export BUILD_ID=1.0.0-alpha-rc1
+export BUILD_NUMBER=20141213
diff --git a/sabresd_6sx/build_id.mk b/sabresd_6sx/build_id.mk
index a9f5b63..5b9d9d2 100644
--- a/sabresd_6sx/build_id.mk
+++ b/sabresd_6sx/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=2.0.0-beta-rc4
-export BUILD_NUMBER=20140815
+export BUILD_ID=1.0.0-alpha-rc1
+export BUILD_NUMBER=20141213
-- 
1.8.0

